vars/Fedora.yml

```diff
@@ -5,3 +5,5 @@ blivet_package_list:
   - libblockdev-lvm
   - libblockdev-mdraid
   - libblockdev-swap
+  - vdo
+  - kmod-kvdo
```

Comment:
This is going to need special attention as well, whether it is to disable these options based on OS or to configure them via DKMS or COPR or similar.

Reply:
Fixed:
Added a support for COPR repositories. 
Now the role checks the var file (see `vars/Fedora.yml`) for COPR repositories. If there is a package required and also present in the list, the role will install packages (as well as package required to use COPR).